tycho-its/src/test/java/org/eclipse/tycho/test/product/ProductProgramArgsTest.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2011 Sonatype Inc. and others.
```

Comment:
headers seems wrong here

Reply:
If you created the file as a whole, you should use the current year and your name here, if you extend a existing file at laest update the year (e.g. `2011, 2023`) and add you to the contributor list.